feat: editorial review: Add documentation for display-mode: picture-in-picture (#32471)
mdn/content: files/en-us/web/css/@media/index.md

- {{cssxref("@media/device-width", "device-width")}} {{deprecated_inline}}
  - : Width of the rendering surface of the output device. Deprecated in Media Queries Level 4.
- {{cssxref("@media/display-mode", "display-mode")}}

>   - : The mode in which an application is being displayed within a device's operating system, for example in `fullscreen` or `picture-in-picture` mode.

Reviewer: IMO "within a device's operating system" is a strange statement. Everything is "within a device's operating system". I would just say:

```suggestion
  - : The mode in which an application is being displayed: for example [fullscreen](/en-US/docs/Web/API/Fullscreen_API) or [picture-in-picture](/en-US/docs/Web/API/Document_Picture-in-Picture_API/) mode.
```
